specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/virtualWan.json

```diff
@@ -1532,7 +1841,13 @@
           },
           "202": {
             "description": "Accepted and the operation will complete asynchronously."
-          }
+          },
+          "default": {
+            "description": "Error.",
+            "schema": {
+              "$ref": "./network.json#/definitions/Error"
```

Comment:
>Error [](start = 51, length = 5)

The `./network.json#/definitions/Error` definition is not the right one. The standard error response documented at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#common-api-response-details](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#common-api-response-details) looks like this:
```
{
  "error": {
    "code": "...",
    "message": "...",
    ....
  }
}
```

But the one you're referencing looks like this.
```
{
  "code": "...",
  "message": "...",
  ....
}
```

Reply:
(This applies to the others as well.)

---
In reply to: [326424772](https://github.com/Azure/azure-rest-api-specs/pull/7238#discussion_r326424772) [](ancestors = 326424772)